src/language/i18n.js

```diff
@@ -34,7 +34,7 @@ const I18n = new class {
   }
 
   isLocaleValid(locale) {
-    return this.locales.includes(locale)
+    return this.locales.includes(locale) && isFunction(this.localeFiles[locale])
```

Comment:
why it couldn't be a function ? if we don't have some locale - it just doesn't exists in the object
also why .includes is used ? does object have this method ? shouldn't it be `.hasOwnProperty` or even `locale in this.locales` ?

Reply:
this.locales is an array here, also I don't see another way error appears except this.locales include locale and this.localeFiles is not

Reply:
compare this.locales and localeFiles. maybe they're out of sync

Reply:
they are synced